infrastructure/sentry-subgraph/schema/events/Referee.graphql

```diff
@@ -77,9 +77,16 @@ type RefereeConfig @entity(immutable: false) {
 type PoolSubmission @entity(immutable: false) {
   id: String!
   challengeId: BigInt!
+  poolAddress: Bytes!
+  challenge: Challenge!
+  poolInfo: PoolInfo!
   stakedKeyCount: BigInt!
   winningKeyCount: BigInt!
   claimedRewardsAmount: BigInt!
-  poolAddress: Bytes!
+  assertionsStateRootOrConfirmData: String!
+  createdTimestamp: BigInt!
```

Comment:
Have we considered if we should add the @Index param to any specific fields that we may want to select/order by in queries to improve query performance?

Reply:
I did not yet consider it, as we did not include the @Index in our Submission entity. But we could add it to the field poolAddress